first-analysis-steps/bookkeeping.md

```diff
@@ -87,7 +86,7 @@ python file”, and add `.py` to the end of the text in “Save As...”. Clicki
 download. Save this file somewhere you can find it again.
 
 A copy of the file we just downloaded is [available
-here](data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09b_Trig0x6138160F_Reco16_Turbo03_Stripping28NoPrescalingFlagged_ALLSTREAMS.DST.py).
+here](data/MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09c_Trig0x6138160F_Reco16_Turbo03_Stripping28r1NoPrescalingFlagged_ALLSTREAMS.DST.py).
```

Comment:
Seeing as we now have #146, is it necessary to download this file any more?

Reply:
Don't we need it so that we can interactively explore our DST?